src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -17,19 +17,29 @@
 """
 
 import logging
+import sys
 from typing import List, Optional, Union
 
 from ...feature_extraction_utils import BatchFeature
 from ...image_utils import ImageInput, is_valid_image
-from ...processing_utils import ProcessorMixin
+from ...processing_utils import (
+    ImagesKwargs,
+    ProcessingKwargs,
+    ProcessorMixin,
+    TextKwargs,
+    _validate_images_text_input_order,
+)
 from ...tokenization_utils_base import (
     AddedToken,
-    PaddingStrategy,
     PreTokenizedInput,
     TextInput,
-    TruncationStrategy,
 )
-from ...utils import TensorType
+
+
+if sys.version_info >= (3, 11):
+    from typing import Unpack
+else:
+    from typing_extensions import Unpack
```

Comment:
If #[31198](https://github.com/huggingface/transformers/pull/31198) is merged in first - then this should be imported from processing_utils cc @molbap as we might need a PR to clean this up for other processors after 31198 merge